### PR TITLE
Improved the "Show Video Resolution on Video start"

### PR DESCRIPTION
**Pull request by frodo19**
Improved the "Show Video Resolution on Video start" Added new images for resolution and audio (flags/ fullscreen folder). Replaced the images to the left bottom corner, cos lot of tv has in upper left, own badges. Also added images to the colored mod too.
Added the movie dvd popup animation when the movie osd open. Replaced the "Show Video Resolution on Video start" setting to the video osd section, because now works in all system, not just in CE.

Also [here is the psd.](https://mega.nz/file/6kV1jKAa#Bm_6OrmFD_QZt5yrirSYSQHUgkiiBOa6rRtbCt0AGGc).

### CODE REVIEW

**Comment by b-jesch** [comment]
Sorry, but there are to much different topics on one pull request. Pls split them in lower parts. Second, I do not agree with some code changes. especially the parts in Variables.xml can be simplified considerably by renaming the flag filenames to  their correspondig code names, e.g. `$INFO[VideoPlayer.AudioCodec,flags/fullscreen/,.png]` and/or renamig special codecs like `DTS-MA-HD.png` to `dtsma.png` in this way: https://github.com/b-jesch/skin.estuary.modv2/blob/8f5f0358176660c2d79b298cb830f1a965754785/xml/Variables.xml#L1721-L1726. Third it makes no sense showing DV or HDR Flags in fullscreen mode if the TV is not capable showing these formats, as we have already discussed here: https://github.com/b-jesch/skin.estuary.modv2/issues/142. Showing DV logo on a non capable TV is simply not the right way.

Also I've decided not to show some background if the content came from a plugin (you have reverted this). Reason for that is here:  https://github.com/b-jesch/skin.estuary.modv2/issues/166. Especially this makes no sense to show 'Master Chief' of 'Halo' when children related content from a KiKa plugin is shown.

Some of your code parts and suggestions I'll adopt with own code changes.

Regards, Happy Christmas and a happy new Year.

File: xml/Variables.xml
Comment (b-jesch) [inline xml/Variables.xml:1]: This has already been implemented...

**Comment by frodo19** [comment]
Hi jesch, 
no problem maybe I used older version of the code?
Just see wht I tryed to  made, and implement with your code.

I will stay open this , you can close when you ready.

Thanks and Merry Christmas and a happy new Year.

**Comment by b-jesch** [comment]
- slide animation to disc rotation in OSD added
- Display of video resolution (HDR, HDR+, HLG, Dolby Vision) and position setting (top/bottom left via setup) improved - CoreElec only atm.
- changing some fullscreen textures against yours

The recognition of the exact video resolution type transmitted via hardware chain is possible only on amlogic devices (I think). Maybe there will be more infolabels later for other Systems (Win, Linux, etc.)

**Comment by b-jesch** [comment]
Thank you very much for contribution. I would like to look over the pull request again next year. Until then, a happy new year.

**Comment by frodo19** [comment]
Vielen Dank für Ihre Arbeit , Guten Rutsch  Frohes neues Jahr !

**Comment by b-jesch** [comment]
Some statistics of the skin from the Kodinerds Repo Admin view. The downloads are really requested by Kodi via "Install from Repo" and show the total amount of Omega ModV2 and from the actual version. But nobody knows if these downloads are really in use of course. ;)

EMV2 Downloads

**Pull request closed without merging**